### PR TITLE
Security guide

### PR DESCRIPTION
**Pull request by funk-yourself**

### CODE REVIEW

**Comment by cveneziani** [comment]
I think it would be nice to also mention what is the default value of each Security option. 
What do you think?

**Comment by funk-yourself** [comment]
@davydovanton @cveneziani done

**Comment by marionschleifer** [comment]
@hanami/core Can we merge this? WDYT?

File: source/guides/1.0/projects/security.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Web applications can send this header to prevent Clickjacking attacks:
+
+```ruby
+# Denies all untrusted domains (default)
+security.x_frame_options 'DENY'
+```
+
+```ruby
+# Allows iframes on example.com
+security.x_frame_options 'http://example.com'
```
Comment (AlfonsoUceda) [inline source/guides/1.0/projects/security.md]: I think the value should be `ALLOW FROM http://example.com`

File: source/guides/head/projects/security.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Web applications can send this header to prevent Clickjacking attacks:
+
+```ruby
+# Denies all untrusted domains (default)
+security.x_frame_options 'DENY'
+```
+
+```ruby
+# Allows iframes on example.com
+security.x_frame_options 'http://example.com'
```
Comment (AlfonsoUceda) [inline source/guides/head/projects/security.md]: I think the value should be `ALLOW FROM http://example.com`

**Comment by funk-yourself** [comment]
@AlfonsoUceda thanks for pointing out the mistake. Now it should be correct.

**Pull request merged**